acceptance/tests/invalid_ssl_config.rb

```diff
@@ -59,8 +59,8 @@
 on agent1, puppet('resource service pxp-agent ensure=running')
 retry_on(agent1, "grep 'TLS handshake failed' #{logfile(agent1)}", {:max_retries => 30,
                                                              :retry_interval => 1})
-assert(on(agent1, "grep 'retrying in' #{logfile(agent1)}"),
-       "pxp-agent should log that it will retry connection")
+retry_on(agent1, "grep 'retrying in' #{logfile(agent1)}", {:max_retries => 10,
```

Comment:
Not sure why we would need to retry here; as the test already waits for the prior log entry to appear. Retrying can only be helpful here though; to avoid spurious fails.
One comment on this is that retry_on means we lose the clearly defined assertion; we could consider a try/rescue with an explicit fail if retry_on fails